std/parallelism.d

```diff
@@ -1067,7 +1067,7 @@ private:
         {
             job.job();
         }
-        catch (Throwable e)
+        catch (Exception e)
```

Comment:
Given the nature of std.parallelism, I think that there's a good chance that it really is supposed to be catching `Throwable` in each of these catches, but I don't know.